data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
+				goto main
+			label chilia
+			`You follow the directions provided you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door "Greetings, captain <first>, we meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`
+			label main
+			`	He leads you to a room that looks like an armory where a team of techs are putting the finishing touches on a rifle stock. "The request from the tech on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since our starships have been brought up to their current capabilities." He gestures, and a tech takes the container with the tentacle from you and starts installing it in a framework.`
+			choice
+				`	"Void rifle?"`
+				`	(expression of boredom)`
+					goto end
+			`	"Yes, void rifle. I don't know who discovered it, but someone figured out that if a particularly high charge was poured through the tentacles nervous system in a specific pattern, it emits a paralyzing cone of energy with a reasonably decent range. I'm not an engineer, so I do not really understand the mechanics of it." He pauses to watch as a tech installs the now-caged tentacle on top of the stock. "Suffice to say, it is useless for sniping, but its ability to shoot through walls and affect areas makes it excellent for clearing ships and tight quarters. Back when we didn't have the ships to resist the Korath, most of our battles were resolved in building to building combat. These rifles gave us the advantage we needed to hold our own until we could even the battlefield up there." He gestures upwards.`
+			label end
+			`	"Enough of history, though." He turns to the techs who are finishing the assembly. "The tentacle has been treated to resist decay and the rifle itself will provide basic maintenance routines for as long as you charge it regularly." He strides over, picks up the completed void rifle, and then hands it to you. "You are holding a piece of our history, Captain. Use it well!"`
```

Comment:
`for as long as you charge it regularly` would seem to imply that the weapon should actually consume some ship energy, but it doesn't (I think that would be really cool).

Reply:
I could definitely add that, although I'm not sure what value would be reasonable.

On the flip side, I"m not sure a unique h2h weapon should be drawing enough power to be significant or noticeable on any ship other than those with the smallest or weakest of generators. Ideally it'd probably be more of a "lump sum" draw after the player has engaged in h2h combat, but we don't have mechanics for that, as far as I can tell.

Reply:
OK, for now I have given it `"Energy Consumption" 0.01` That's quite low - probably insignificant, but it is there.